### ISSUE TITLE
Release v0.0.1-alpha

### ISSUE THREAD

**Issue by Keith-CY** [ckb-js/kuai#360]
An alpha release could be published for external teams to have a try after

- https://github.com/ckb-js/kuai/pull/319
- https://github.com/ckb-js/kuai/pull/341

Are merged

Before release, the following should be checked

1. version number
2. release note
3. package namespace
4. release workflow

**Comment by Keith-CY** [comment]
For simplicity, `version number` and `release note` will be generated automatically by `lerna` in an github workflow.

So the workflow would be

1. run tests;
2. bump versions by some rule, probably the name of branch;
3. update changelog;
4. open a PR to the `main` branch;
5. create a release once the PR is merged.

**Comment by Keith-CY** [comment]
I'm adding the GitHub action to update versions and changelogs, blocked by an issue that `lerna version` is executed multiple times without specifying an explicit `version number`, still WIP.

**Comment by Keith-CY** [comment]
> For simplicity, `version number` and `release note` will be generated automatically by `lerna` in an github workflow.
> 
> So the workflow would be
> 
> ### workflow 1
> 1. run tests;
> 2. bump versions by some rule, probably the name of branch;
> 3. update changelog;
> 4. open a PR to the `main` branch;
> 
> ### workflow 2
> 5. create a release once the PR is merged.

During this work, I would divide these 2 workflows into 3

1. workflow 1: triggered by new branch of `rc/v*` and open a PR to `rc/v*` for bumping versions and changelogs
2. workflow 2: triggered by merge of the PR opened in workflow 1, open a PR from `rc/v*` to `main`
3. workflow 3: triggered by merge of the PR opened in workflow 2, create a release when `rc/v*` is merged into `main`

**Comment by Keith-CY** [comment]
Released at https://github.com/ckb-js/kuai/releases/tag/v0.0.1-alpha
